### ISSUE TITLE
Error when calling the `update_rates` method

### ISSUE THREAD

**Issue by cristielciu** [phlegx/money-currencylayer-bank#10]
I'm getting this error when I try to call `update_rates` with a valid API Access Key
```ruby
Traceback (most recent call last):
        1: from (irb):24
Money::Bank::Error (You have not supplied a valid API Access Key. [Technical Support: support@apilayer.com])
```

I think this is due to a change in the currency layer API endpoint
More details [here](https://apilayer.com/marketplace/currency_data-api)

If my investigation is right then the fix should be a quick one
Just update https://github.com/phlegx/money-currencylayer-bank/blob/master/lib/money/bank/currencylayer_bank.rb#L34 with 

```
CL_URL = 'api.apilayer.com/currency_data/live'.freeze
```

and https://github.com/phlegx/money-currencylayer-bank/blob/master/lib/money/bank/currencylayer_bank.rb#L243 with 

```
URI.open(source_url, "apikey" => access_key).read
```

**Comment by phlegx** [comment]
Like described in the documentation, the endpoint is correct: https://currencylayer.com/documentation

See my last commit in master. I have changed `http` to `https`. Can you give it a try please?

**Comment by phlegx** [comment]
This gem works only with API access keys created on https://currencylayer.com. New created API access keys from https://apilayer.com do not work with this gem. I will release a new version that support both endpoints.

**Comment by blazeblazeblaze** [comment]
Hi @phlegx . We're getting `Money::Bank::Error: Access Restricted - Your current Subscription Plan does not support HTTPS Encryption.` errors when using a free plan. The assumption is that the free plan doesn't support HTTPs, perhaps this should be configurable. In a meantime, we downgraded which resolved this issue.

**Comment by pradosh987** [comment]
Hi @phlegx 
Shouldn't it be `http` as default value for `CL_URL` which was changed in this commit https://github.com/phlegx/money-currencylayer-bank/commit/63d86f750849c3fd2bd2a4f5f83ff519fbe2be05. Otherwise setting `secure_connection = false` will get ignored and this will have no effect because CL_URL always starts with `https`

```
 CL_URL = 'https://api.currencylayer.com/live'.freeze
  # CurrencylayerBank secure url
  CL_SECURE_URL = CL_URL.sub('http:', 'https:')     
  ```

**Comment by phlegx** [comment]
@blazeblazeblaze @pradosh987 thats true. I rewrite the gem to support the new and the old endpoints soon as possible.

**Comment by phlegx** [comment]
**Released v0.7.0** 🎉
```ruby
# (optional)
# Use the old endpoint api.currencylayer.com. By default, the new endpoint is used.
mclb.currencylayer = true
```
